ConfigCI PowerShell reference update

TimShererWithAquent opened the pull request:
[1880232](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1880232)

New cmdlet: Set-CIPolicySetting
Changes to 6 others

TimShererWithAquent committed:
Add changes from PlayPS.

opbld31 commented:
Docs Build status updates of commit _[da23325](https://github.com/TimShererWithAquent/windows-powershell-docs/commits/da2332538ede28fe55d1fcb34f3d7cccc8dfea14)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docset/winserver2022-ps/configci/Add-SignerRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/configci-v-tishe/docset/winserver2022-ps/configci/Add-SignerRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/configci/add-signerrule?view=windowsserver2022-ps&branch=pr-en-us-2671) (WindowsServer2022-ps) |
[docset/winserver2022-ps/configci/ConfigCI.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/configci-v-tishe/docset/winserver2022-ps/configci/ConfigCI.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/configci/index?view=windowsserver2022-ps&branch=pr-en-us-2671) (WindowsServer2022-ps) |
[docset/winserver2022-ps/configci/Edit-CIPolicyRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/configci-v-tishe/docset/winserver2022-ps/configci/Edit-CIPolicyRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/configci/edit-cipolicyrule?view=windowsserver2022-ps&branch=pr-en-us-2671) (WindowsServer2022-ps) |
[docset/winserver2022-ps/configci/Merge-CIPolicy.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/configci-v-tishe/docset/winserver2022-ps/configci/Merge-CIPolicy.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/configci/merge-cipolicy?view=windowsserver2022-ps&branch=pr-en-us-2671) (WindowsServer2022-ps) |
[docset/winserver2022-ps/configci/New-CIPolicy.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/configci-v-tishe/docset/winserver2022-ps/configci/New-CIPolicy.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/configci/new-cipolicy?view=windowsserver2022-ps&branch=pr-en-us-2671) (WindowsServer2022-ps) |
[docset/winserver2022-ps/configci/New-CIPolicyRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/configci-v-tishe/docset/winserver2022-ps/configci/New-CIPolicyRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/configci/new-cipolicyrule?view=windowsserver2022-ps&branch=pr-en-us-2671) (WindowsServer2022-ps) |
[docset/winserver2022-ps/configci/Set-CIPolicyIdInfo.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/configci-v-tishe/docset/winserver2022-ps/configci/Set-CIPolicyIdInfo.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/configci/set-cipolicyidinfo?view=windowsserver2022-ps&branch=pr-en-us-2671) (WindowsServer2022-ps) |
[docset/winserver2022-ps/configci/Set-CIPolicySetting.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/configci-v-tishe/docset/winserver2022-ps/configci/Set-CIPolicySetting.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/configci/set-cipolicysetting?view=windowsserver2022-ps&branch=pr-en-us-2671) (WindowsServer2022-ps) |
[docset/winserver2022-ps/configci/Set-HVCIOptions.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/configci-v-tishe/docset/winserver2022-ps/configci/Set-HVCIOptions.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/configci/set-hvcioptions?view=windowsserver2022-ps&branch=pr-en-us-2671) (WindowsServer2022-ps) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C23%5C46a32786-a1f6-1250-e1e1-2a4554025dc9%5CPullRequest%5C202109231741409664-2671%5Cworkflow_report.html?sv=2020-08-04&se=2021-10-24T17%3A48%3A37Z&sr=b&sp=r&sig=wyljX%2FaQeY%2BORKNY4LsC%2BKyvUlVT6d02%2Bpxq5XluUVM%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/46a32786-a1f6-1250-e1e1-2a4554025dc9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

TimShererWithAquent committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/windows-powershell-docs into configci-v-tishe

TimShererWithAquent committed:
Incorporate feedback.

TimShererWithAquent committed:
Add link.

TimShererWithAquent committed:
Additional fix.

TimShererWithAquent committed:
Merge branch 'master' into configci-v-tishe

TimShererWithAquent committed:
Fix link.

We're holding pub on this one until this PR is published so we can link to it, per request: MicrosoftDocs/windows-docs-pr/pull/5847

TimShererWithAquent committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/windows-powershell-docs into configci-v-tishe

TimShererWithAquent committed:
Fix link.

TimShererWithAquent committed:
Merge branch 'configci-v-tishe' of https://github.com/TimShererWithAquent/windows-powershell-docs into configci-v-tishe

TimShererWithAquent committed:
Remove link.

TimShererWithAquent committed:
Merge branch 'master' into configci-v-tishe

JohanFreelancer9 commented on docset/winserver2022-ps/configci/New-CIPolicy.md:
```diff
@@ -407,6 +475,22 @@ Accept pipeline input: False
 
+### -MultiplePolicyFormat
+Indicates that this cmdlet should create a policy in multiple policy format as opposed to a single policy format. 
+Refer to [Create WDAC policies in Multiple Policy Format](https://docs.microsoft.com/windows/security/threat-protection/windows-defender-application-control/deploy-multiple-windows-defender-application-control-policies#creating-wdac-policies-in-multiple-policy-format) for the difference between the policy formats. 
```
```suggestion
Refer to [Create WDAC policies in Multiple Policy Format](/windows/security/threat-protection/windows-defender-application-control/deploy-multiple-windows-defender-application-control-policies#creating-wdac-policies-in-multiple-policy-format) for the difference between the policy formats. 
```

JohanFreelancer9 commented on docset/winserver2022-ps/configci/New-CIPolicy.md:
```diff
@@ -525,6 +609,22 @@ Accept pipeline input: False
 
+### -SpecificFileNameLevel
+Specifies the attribute of the file off which to base a file name rule. The -Level must be set to FileName for this option.
+Refer to [File Name Rules Info](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create#windows-defender-application-control-filename-rules) for a description of the acceptable values.
```
```suggestion
Refer to [File Name Rules Info](/windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create#windows-defender-application-control-filename-rules) for a description of the acceptable values.
```

JohanFreelancer9 commented on docset/winserver2022-ps/configci/New-CIPolicyRule.md:
```diff
@@ -243,6 +246,43 @@ This set of commands finds a packaged application matching the specified name an
+Specifies an app.
+This cmdlet creates per-app rules which control whether specific plug-ins, add-ins, and modules can run from specific apps.
+
+For more information, see [Use a Windows Defender Application Control policy to control specific plug-ins, add-ins, and modules](https://docs.microsoft.com/windows/security/threat-protection/windows-defender-application-control/use-windows-defender-application-control-policy-to-control-specific-plug-ins-add-ins-and-modules).
```
```suggestion
For more information, see [Use a Windows Defender Application Control policy to control specific plug-ins, add-ins, and modules](/windows/security/threat-protection/windows-defender-application-control/use-windows-defender-application-control-policy-to-control-specific-plug-ins-add-ins-and-modules).
```

JohanFreelancer9 commented on docset/winserver2022-ps/configci/New-CIPolicyRule.md:
````diff
@@ -314,23 +354,22 @@ This cmdlet will not check whether the filepath string is a valid filepath.
 ```
 
 ### -Level
 Specifies the primary level of detail for generated rules. Refer to [WDAC File Rule Levels](https://docs.microsoft.com/windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create#windows-defender-application-control-file-rule-levels) for acceptable parameter values and descriptions.
````
```suggestion
Specifies the primary level of detail for generated rules. Refer to [WDAC File Rule Levels](/windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create#windows-defender-application-control-file-rule-levels) for acceptable parameter values and descriptions.
```

JohanFreelancer9 commented on docset/winserver2022-ps/configci/New-CIPolicyRule.md:
```diff
@@ -371,11 +410,11 @@ Accept wildcard characters: False
-Specifies the attribute of the file off which to base a file name rule. The -Level must be set to FileName for this option. 
-Refer to [File Name Rules Info](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create#windows-defender-application-control-filename-rules) for a description of the acceptable values. 
+Specifies the attribute of the file off which to base a file name rule. The -Level must be set to FileName for this option.
+Refer to [File Name Rules Info](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create#windows-defender-application-control-filename-rules) for a description of the acceptable values.
```
```suggestion
Refer to [File Name Rules Info](/windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create#windows-defender-application-control-filename-rules) for a description of the acceptable values.
```

JohanFreelancer9 reviewed (changes requested):
Copy looks good. However, I have suggested changing the links to site relative links, unless there is a reason not to. @TimShererWithAquent, @dariomws, @get-itips, @tseward. Many thanks.

dariomws reviewed (approved):
I'm ok after suggestions.

TimShererWithAquent committed:
Apply suggestions from code review

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>
Co-authored-by: Dario Woitasen <33589238+dariomws@users.noreply.github.com>

TimShererWithAquent commented:
@JohanFreelancer9 Thanks - I rechecked the style guide and site relative links are correct, as you said. Accepted your changes.

opbld32 commented:
Docs Build status updates of commit _[dedf295](https://github.com/TimShererWithAquent/windows-powershell-docs/commits/dedf295ffae3873dcb4fa96ff302caaaac34b1e0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docset/winserver2022-ps/configci/Add-SignerRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/configci-v-tishe/docset/winserver2022-ps/configci/Add-SignerRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/configci/add-signerrule?view=windowsserver2022-ps&branch=pr-en-us-2671) (WindowsServer2022-ps) |
[docset/winserver2022-ps/configci/ConfigCI.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/configci-v-tishe/docset/winserver2022-ps/configci/ConfigCI.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/configci/index?view=windowsserver2022-ps&branch=pr-en-us-2671) (WindowsServer2022-ps) |
[docset/winserver2022-ps/configci/Edit-CIPolicyRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/configci-v-tishe/docset/winserver2022-ps/configci/Edit-CIPolicyRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/configci/edit-cipolicyrule?view=windowsserver2022-ps&branch=pr-en-us-2671) (WindowsServer2022-ps) |
[docset/winserver2022-ps/configci/Merge-CIPolicy.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/configci-v-tishe/docset/winserver2022-ps/configci/Merge-CIPolicy.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/configci/merge-cipolicy?view=windowsserver2022-ps&branch=pr-en-us-2671) (WindowsServer2022-ps) |
[docset/winserver2022-ps/configci/New-CIPolicy.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/configci-v-tishe/docset/winserver2022-ps/configci/New-CIPolicy.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/configci/new-cipolicy?view=windowsserver2022-ps&branch=pr-en-us-2671) (WindowsServer2022-ps) |
[docset/winserver2022-ps/configci/New-CIPolicyRule.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/configci-v-tishe/docset/winserver2022-ps/configci/New-CIPolicyRule.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/configci/new-cipolicyrule?view=windowsserver2022-ps&branch=pr-en-us-2671) (WindowsServer2022-ps) |
[docset/winserver2022-ps/configci/Set-CIPolicyIdInfo.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/configci-v-tishe/docset/winserver2022-ps/configci/Set-CIPolicyIdInfo.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/configci/set-cipolicyidinfo?view=windowsserver2022-ps&branch=pr-en-us-2671) (WindowsServer2022-ps) |
[docset/winserver2022-ps/configci/Set-CIPolicySetting.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/configci-v-tishe/docset/winserver2022-ps/configci/Set-CIPolicySetting.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/configci/set-cipolicysetting?view=windowsserver2022-ps&branch=pr-en-us-2671) (WindowsServer2022-ps) |
[docset/winserver2022-ps/configci/Set-HVCIOptions.md](https://github.com/TimShererWithAquent/windows-powershell-docs/blob/configci-v-tishe/docset/winserver2022-ps/configci/Set-HVCIOptions.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/configci/set-hvcioptions?view=windowsserver2022-ps&branch=pr-en-us-2671) (WindowsServer2022-ps) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C11%5C1%5C46a32786-a1f6-1250-e1e1-2a4554025dc9%5CPullRequest%5C202111011436258648-2671%5Cworkflow_report.html?sv=2020-08-04&se=2021-12-02T14%3A43%3A33Z&sr=b&sp=r&sig=fMnOtJEQGA5fBWjkUExF9tGyvzDIAYIL5J6zYBc1OmI%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/46a32786-a1f6-1250-e1e1-2a4554025dc9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

TimShererWithAquent commented:
Hi @tiburd - I've accepted the changes from the review. This one should be ready to merge.
